frontend/packages/data-portal/app/utils/featureFlags.ts

```diff
@@ -0,0 +1,45 @@
+import { useSearchParams } from '@remix-run/react'
+
+import { useEnvironment } from 'app/context/Environment.context'
+
+export type FeatureFlagEnvironment = typeof process.env.ENV
+
+export const FEATURE_FLAGS = {
+  methodType: 'dev',
+} as const
+
+export type FeatureFlagKey = keyof typeof FEATURE_FLAGS
+
+const ENABLE_FEATURE_PARAM = 'enable-feature'
+const DISABLE_FEATURE_PARAM = 'disable-feature'
+
+export function getFeatureFlag({
```

Comment:
this seems more like a user setting than anything – would we be accomplishing this stuff with cookies in the future?